internal/command/command.go

```diff
@@ -424,14 +431,48 @@ func UsageMetrics(command *cobra.Command, wg *sync.WaitGroup) {
 
 // GlobalFlags contains all global flags definitions.
 type GlobalFlags struct {
-	Filter           string
-	Format           string
-	Save             string
-	Host             string
-	HostNetworkKey   string
-	Log              string
-	Network          string
-	Yes              bool
-	ConfigPaths      []string
-	SkipVersionCheck bool
+	Filter                     string
+	Format                     string
+	Save                       string
+	Host                       string
+	HostNetworkKey             string
+	Log                        string
+	Network                    string
+	Yes                        bool
+	ConfigPaths                []string
+	SkipVersionCheck           bool
+	SkipContractMigrationCheck bool
+}
+
+
+const migrationDataURL = "https://github.com/onflow/cadence/tree/master/migrations_data"
+
+func checkContractMigrations(state *flowkit.State, logger output.Logger, flow flowkit.Services) {
+	contractStatuses, err := validator.NewValidator(github.NewClient(nil).Repositories, flow.Network(), state, logger).GetContractStatuses()
+	if err != nil {
+		logger.Error(fmt.Sprintf("failed to validate contracts: %s", err))
```

Comment:
This should just be ignored.  If we can't get the migration report, we shouldn't log anything to the user.  This is something that happens silently in the background unrelated to their actions in the Flow CLI, and only notifies if a failed migration is found, so I think logging this causes more harm than good

Reply:
updated, thanks!